frontend/packages/pipelines-plugin/src/components/repository/sections/RepositoryOverview.tsx

```diff
@@ -88,7 +98,23 @@ const RepositoryOverview = () => {
           {t('pipelines-plugin~Commit these changes and push them to your Git repository.')}
         </Text>
       </FormGroup>
-      <FormGroup fieldId="step-4">
+      {!(values.gitProvider === GitProvider.GITHUB && values.method === GitProvider.GITHUB) && (
```

Comment:
This check is not correct. I could not see the webhook URL when creating a PaC Repo with a GitHub webhook.

You want to show this always when using Webhooks, also when using GitHub. Only in the case when you use the GitHub **_App_** flow we should not show the webhook. @vikram-raj can help here next week if the difference isn't clear.

And this section should be also skipped when `values?.webhook?.url` is not defined.

Reply:
Hi @jerolimov ,

This is know issue and I have PR up for this already - https://github.com/openshift/console/pull/12593. If you don't enter access token then webhook information is not getting saved, so we decided to make git access token as mandatory. 

if it is only github provider and Use GitHub app is selected, then only we don't need to show webhook URL, for remaining cases we have webhook URL visible for the users. 

Please correct me if I am wrong.